### PR TITLE
justify content and fix max width

### PR DESCRIPTION
**Pull request by mukeshkharita**

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -115,7 +116,7 @@ blockquote p {
   margin-left: auto;
   margin-right: auto;
   /* max-width: 1368px; */
```
Comment (paramsingh) [inline css/main.css:118]: can we remove this comment?

**Review by paramsingh — CHANGES_REQUESTED** [review]

File: css/main.css (hunk 1 above)
Comment (mukeshkharita) [reply css/main.css:118]: TBH no, this is for reference,

File: css/main.css [hunk 2]
```diff
@@ -8,7 +8,8 @@ body {
   color: #515151;/*#3A3A3A;/*rgba(0,0,0,.8);/*#666;*/
   word-wrap: break-word;
   line-height: 1.75rem;
-  text-align: justified;
+  text-align: justify;
+  text-justify: inter-word;
```
Comment (vansika) [inline css/main.css:12]: Afaik, you said this isn't required in the body tag, but good to have I guess.

**Review by vansika — APPROVED** [review]

File: css/main.css (hunk 1 above)
Comment (vansika) [reply css/main.css:118]: You can totally ignore @paramsingh

**Review by paramsingh — APPROVED** [review]

**Pull request merged**